model/patch_lifecycle.go

```diff
@@ -70,8 +87,8 @@ func ValidateTVPairs(p *Project, in []TVPair) error {
 // Given a patch version and a list of variant/task pairs, creates the set of new builds that
 // do not exist yet out of the set of pairs. No tasks are added for builds which already exist
 // (see AddNewTasksForPatch).
-func AddNewBuildsForPatch(p *patch.Patch, patchVersion *version.Version, project *Project, pairs TVPairSet) error {
-	taskIds := NewPatchTaskIdTable(project, patchVersion, pairs)
+func AddNewBuildsForPatch(p *patch.Patch, patchVersion *version.Version, project *Project, tasks TaskVariantPairs) error {
+	taskIds := NewPatchTaskIdTable(project, patchVersion, tasks) // TODO: handle display tasks
```

Comment:
is this todo still relevant?

Reply:
yep, but it was resolved with the commit right before "fix updating of patches"